packages/odyssey-react/src/components/Heading/Heading.module.scss

```diff
@@ -12,58 +12,58 @@
 
 .root {
   margin-block-start: 0;
-  margin-block-end: $spacing-xs-em;
+  margin-block-end: var(--MarginBlockEnd);
   margin-inline: 0;
-  color: $text-heading;
-  font-family: $body-font-family;
-  font-size: $base-font-size;
+  color: var(--TextColor);
+  font-family: var(--FontFamily);
+  font-size: var(--FontSize);
   font-weight: 600;
 }
 
 .level1 {
-  font-size: $size-heading-1;
+  font-size: var(--Level1FontSize);
 }
 
 .level2 {
-  font-size: $size-heading-2;
+  font-size: var(--Level2FontSize);
 }
 
 .level3 {
-  font-size: $size-heading-3;
+  font-size: var(--Level3FontSize);
 }
 
 .level4 {
-  font-size: $size-heading-4;
+  font-size: var(--Level4FontSize);
 }
 
 .level5 {
-  font-size: $size-heading-5;
+  font-size: var(--Level5FontSize);
 }
 
 .level6 {
-  font-size: $size-heading-6;
+  font-size: var(--Level6FontSize);
 }
 
 .level1,
 .level2,
 .level3 {
-  line-height: $heading-line-height;
+  line-height: var(--Heading);
```

Comment:
this doesn't align with our naming standards! we'll want something like `HeadingLineHeight` here and critically, re-use that same value below on line 68.

The same pattern will hold for what we're calling `lineHeight="base"` in props as well. line 56 below should be `BaseLineHeight` and we would re-use that same token on line 64 as well.

The structure of these styles and the fact that line heights can be overriden explicitly, and are also applied implicitly based on heading level makes this more complicated than it normally is